sos/archive.py

```diff
@@ -209,8 +210,15 @@ def add_node(self, path, mode, device):
         dest = self.dest_path(path)
         self._check_path(dest)
         if not os.path.exists(dest):
-            os.mknod(dest, mode, device)
-            shutil.copystat(path, dest)
+            try:
+                os.mknod(dest, mode, device)
+                shutil.copystat(path, dest)
```

Comment:
The `copystat` doesn't really need to be under the try, but it's not a big deal.